src/customer/CustomerState.php

```diff
@@ -0,0 +1,65 @@
+<?php declare(strict_types=1);
+
+namespace hiqdev\php\billing\customer;
+
+class CustomerState
```

Comment:
Should be converted to Enum, probably

Reply:
I was Enum, but since I added CustomerState into Customer class it caused error in Hydrator, thus I changed CustomerState from Enum to class